competition/human_evaluation/human-evaluation-results.py

```diff
@@ -0,0 +1,261 @@
+#!/usr/bin/env python
+import click
+import logging
+import numpy as np
+import os
+import pandas as pd
+
+from collections import Counter, defaultdict
+from copy import deepcopy
+
+SEED = 2019
```

Comment:
Maybe add a comment here that indicates that this seed was also used for data _collection_ and therefore cannot be changed?

Reply:
good call, done